src/Umbraco.Cms.Api.Management/ViewModels/Document/DocumentUrlInfoResourceSet.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Umbraco.Cms.Api.Management.ViewModels.Document;
+
+public sealed class DocumentUrlInfoResourceSet
```

Comment:
I'm unsure about naming here, as our usual naming is either suffixed with `ViewModel` or `ResponseModel` 🤔 
Maybe we should discuss this naming with the team, just to align 😁